Makefile

```
@@ -40,7 +40,7 @@ check-reqs:
 
 ## This function is specific to Jenkins infrastructure and isn't required in other contexts
 docker-init: check-reqs
-	@set -x; docker buildx create --use
+	@set -x; docker buildx create --use --bootstrap --name docker-ssh-agent --driver docker-container --config /etc/buildkitd.toml
```

```suggestion
ifeq ($(CI),true)
	docker buildx create --use --bootstrap --driver docker-container --config /etc/buildkitd.toml
else
	docker buildx create --use --bootstrap --driver docker-container
endif
```
- set config only on CI
- no name
- remove `set -x` and `@` (which were cancelling themselves)